extensions/amp-story/1.0/amp-story-grid-layer.js

```diff
@@ -77,6 +78,26 @@ export const GRID_LAYER_TEMPLATE_CLASS_NAMES = {
   'thirds': 'i-amphtml-story-grid-template-thirds',
 };
 
+/**
+ * The attribute name for grid layer presets.
+ * @private @const {string}
+ */
+const PRESET_ATTRIBUTE_NAME = 'preset';
+
+/**
+ * The attributes that will be applied for each preset.
+ * @private @const {!Object<string, !Object<string, string>>}
+ */
+const GRID_LAYER_PRESETS_ATTRIBUTES = {
```

Comment:
enums are usually singular

Reply:
This is not a enum, it's a dictionary of attributes. I'm following the same naming convention to `GRID_LAYER_TEMPLATE_CLASS_NAMES` by changing it to `GRID_LAYER_PRESET_ATTRIBUTES`, so that should make sense